nistats/datasets.py

```diff
@@ -339,22 +339,22 @@ def _glob_spm_auditory_data():
 
         # volumes for this dataset of shape (64, 64, 64, 1); let's fix this
         for x in _subject_data["func"]:
-            vol = nibabel.load(x)
+            vol = nib.load(x)
             if len(vol.shape) == 4:
-                vol = nibabel.Nifti1Image(vol.get_data()[:, :, :, 0],
-                                          vol.get_affine())
-                nibabel.save(vol, x)
+                vol = nib.Nifti1Image(vol.get_data()[:, :, :, 0],
+                                      vol.affine)
+                vol.to_filename(x)
 
         _subject_data["anat"] = [subject_data[x] for x in subject_data.keys()
                                  if re.match("^sM00223_002\.img$",
                                              os.path.basename(x))][0]
 
         # ... same thing for anat
-        vol = nibabel.load(_subject_data["anat"])
+        vol = nib.load(_subject_data["anat"])
         if len(vol.shape) == 4:
-            vol = nibabel.Nifti1Image(vol.get_data()[:, :, :, 0],
-                                      vol.get_affine())
-            nibabel.save(vol, _subject_data["anat"])
+            vol = nib.Nifti1Image(vol.get_data()[:, :, :, 0],
+                                  vol.affine)
+            vol.to_filename(_subject_data["anat"])
```

Comment:
Here (and above) you may just want [`nib.squeeze_image`](https://github.com/nipy/nibabel/blob/2139ce0d24e65a83295bb6b3eaaf005eaeaebb5f/nibabel/funcs.py#L17-L87).

Reply:
All right, thx for the tip.